public/components/common/modules/panel/components/agg-table.tsx

```diff
@@ -56,18 +56,29 @@ export const AggTable = ({
       },
     };
   };
-
+  const sorting = {
```

Comment:
Adding sorting to an EuiInMemoryTable without knowing that all the possible data is already in memory can lead to unaccurate results.
You could use `onTableChange` as [documented by Eui](https://elastic.github.io/eui/#/tabular-content/in-memory-tables) to change the value of `preAppliedAggs.buckets.terms.order` for either `_count` or `_key` to be `asc` or `desc` as described in the [Elastic doc](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#search-aggregations-bucket-terms-aggregation-order). It would just require mapping the `field` value to one of those.